tests/src/test/scala/pathy/PrintPathSpecs.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2014 - 2015 SlamData Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pathy
+
+import org.specs2.mutable.Specification
+import pathy.Path._
+
+class PrintPathSpecs extends Specification {
```

Comment:
Ah, I thought these were tests for printing, but that is not clear to me anymore. Happy to put them back where they were. But that other file is getting pretty big, we should think about splitting it up a bit.